react-gbajs/src/emulator/video/software.js

```diff
@@ -2,6 +2,14 @@ function MemoryAligned16(size) {
 	this.buffer = new Uint16Array(size >> 1);
 };
 
+// MemoryAligned16.prototype.freeze = function() {
+// 	return cloneDeep(this.buffer);
+// };
+//
+// MemoryAligned16.prototype.defrost = function(frost) {
+// 	this.buffer = frost;
+// };
```

Comment:
oops, forgot to remove this. MemoryAligned16 does not need its own freeze/defrost methods, because both of its subclasses, GameBoyAdvanceOAM and GameBoyAdvanceVRAM have their own freeze/defrost implementations.